tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -5990,6 +5990,7 @@ TEST_P(OpConverter_FP32_Test, ConvertUnary) {
   ADD_OP("Cos", ops::Cos, std::cos);
   ADD_OP("Cosh", ops::Cosh, std::cosh);
   ADD_OP("Exp", ops::Exp, std::exp);
+  ADD_OP("Erf", ops::Erf, std::erf);
```

Comment:
Need to protect this.
Also need to fix these two lines:
https://github.com/tensorflow/tensorflow/blob/9e26f3b56acbff18f01da8afce6b571739f59ef3/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc#L684
https://github.com/tensorflow/tensorflow/blob/9e26f3b56acbff18f01da8afce6b571739f59ef3/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc#L719